src/create_disk.sh

```diff
@@ -123,6 +123,22 @@ set prefix=($prefix)/grub2
 normal
 EOF
 
+# copy the boot binaries needed for Raspberry pi support
+if  [ -d "/usr/sihare/uboot/" ]; then
+	if [ "$arch" == "aarch64" ]; then
+		cp -P /boot/efi/*.bin rootfs/boot/efi/
+		cp -P /boot/efi/*.dat rootfs/boot/efi/
+		cp -P /boot/efi/*.dtb rootfs/boot/efi/
+		cp -P /boot/efi/*.elf rootfs/boot/efi/
+		cp -rP /boot/efi/overlays/ rootfs/boot/efi/
+		cp -P /usr/share/uboot/rpi_3/u-boot.bin rootfs/boot/efi/rpi3-u-boot.bin
+		# rpi4 is only supported in Fedora 31 and on
+		if [ -f "/usr/share/uboot/rpi_4/u-boot.bin" ]; then
```

Comment:
This one is trickier as it's acting as a version conditional, but we could do instead:
```
osverid=(. /etc/os-release && echo "${VERSION_ID}")
if test "$((${osverid} >= 31))" = 1; then
   ...
```